scripts/zones/Ceizak_Battlegrounds/mobs/Transcendent_Scorpion.lua

```diff
@@ -0,0 +1,62 @@
+-----------------------------------
+-- Area: Ceizak Battlegrounds
+-- NM: Transcendent Scorpion
+-----------------------------------
+local entity = {}
+local lowSkill = false
+
+local removables = {
```

Comment:
What's the deal with these large tables of status effects? Two of the are the same (starts with Flash, ends with Max HP Down), and the other is different. Are these certain families of buffs? Should this logic be moved out into a helper, or down into Core?

Reply:
ill be honest, i dont know how to call 2 types of debuffs